.github/workflows/integration-tests-smoke.yml

```diff
@@ -147,9 +149,39 @@ jobs:
         uses: cachix/install-nix-action@29bd9290ef037a3ecbdafe83cbd2185e9dd0fa0a # v20
         with:
           nix_path: nixpkgs=channel:nixos-unstable
+      - name: Setup Scarb for Linux
+        id: install-scarb
+        shell: bash
+        run: |
+          wget https://github.com/software-mansion/scarb/releases/download/${{ env.scarb_version }}/scarb-${{ env.scarb_version }}-x86_64-unknown-linux-musl.tar.gz
+          tar -xvzf scarb-${{ env.scarb_version }}-x86_64-unknown-linux-musl.tar.gz
+          mv -vf scarb-${{ env.scarb_version }}-x86_64-unknown-linux-musl scarb-build
+          echo "$GITHUB_WORKSPACE/scarb-build/bin" >> $GITHUB_PATH
```

Comment:
Why not just use the action we use on all the other workflows

https://github.com/smartcontractkit/chainlink-starknet/blob/57cedea9c926a041ab341839311cc3dac57362c1/.github/workflows/contracts.yml#L41-L42